src/main/java/org/ow2/proactive/connector/iaas/cloud/provider/jclouds/openstack/OpenstackUtil.java

```diff
@@ -126,4 +128,23 @@ public void validateOpenstackInstanceParameters(Instance instance) {
         Validate.notNull(instance.getNumber(), "Instance parameter 'number' cannot be null");
         Validate.notBlank(instance.getNumber(), "Instance parameter 'number type' cannot be empty");
     }
+
+    /**
+     * @return The Operating system family either by default from jclouds or from the user metadata,
+     * created to patch the limitation caused by Jclouds for OpenStack (not taking the distro_family into account).
+     */
+    public static OsFamily getOpenStackOSFamily(Image image) {
+        if (image.getOperatingSystem().getFamily() == org.jclouds.compute.domain.OsFamily.UNRECOGNIZED) {
+            if (image.getUserMetadata().containsKey("distro_family")) {
+                return org.jclouds.compute.domain.OsFamily.fromValue(image.getUserMetadata().get("distro_family"));
```

Comment:
Same for here you can import OsFamily as a class header.

Reply:
noted! ;)

Reply:
noted ;)

Reply:
done